blog/_posts/2025-04-21-vm-generations-and-aks.md

```diff
@@ -1 +1,106 @@
+---
+title: "VM Generations and AKS"
+description: "Learn more about Gen 1 and Gen 2 VMs and what they offer, differences between them, upcoming Generation 1 VM retirements, and how to move your AKS workloads to Generation 2 VMs."
+date: 2025-04-21 # date is important. future dates will not be published
+authors: 
+   - Ally Ford
+   - Sarah Zhou
+   - Jack Jiang
+categories: 
+- operations
+---
+
+## What are Virtual Machine Generations?
+
+Azure virtual machine sizes are broken down by the amount/type of resources allocated to the virtual machine in the cloud. These resources are in turn 
+tied to a portion of a physical server's hardware capabilities. A single physical server may be broken down into a collection of *size series*, or 
+configurations available utilizing its resources. 
+
+As the physical hardware ages and newer components become available, older hardware of a previous generation get retired, while newer generation hardware 
+is progressively made available. 
+
+In this blog, we will go over Generation 1 and the newer Generation 2 virtual machines. Both have their own use cases, and [picking the right one](https://learn.microsoft.com/en-us/windows-server/virtualization/hyper-v/plan/should-i-create-a-generation-1-or-2-virtual-machine-in-hyper-v) 
```

Comment:
We should be careful here in saying "both have their own use cases". In reality we are working on getting rid of Gen 1 completely